tests/ports/psoc6/run_psoc6_tests.sh

```diff
@@ -236,6 +237,10 @@ i2s_tests() {
   "" "i2s_tx" ${dev_stub} "${tests_psoc6_dir}/board_ext_hw/multi/i2s_tx.py"
 }
 
+pdm_pcm_tests() {
+  run_tests "pdm_pcm" ${dev_test} "${tests_psoc6_dir}/board_only_hw/single/pdm_pcm.py"
```

Comment:
As this is in the board_only_hw it is still running that group in the ci/cd. Not as a separate test.
As said, I would add it to the ci-tests function.
And while you are focusing on this module development, remember that for this branch you can change the workflow to only run this test. And make a faster ci/cd only running this test, but also for other boards that the one you are not using in your desktop. Not necessary, but might help.
Then before merging, we can enable all the rest and worry a bit of anything we have broken (like I2S).

I will also try to fix (tomorrow?) the 063-BLE which is now recurrently happening.